network/wsNetwork.go

```diff
@@ -1553,6 +1583,11 @@ func (wn *WebsocketNetwork) tryConnect(addr, gossipAddr string) {
 	defer wn.wg.Done()
 	requestHeader := make(http.Header)
 	wn.setHeaders(requestHeader)
+	for _, supportedProtocolVersion := range SupportedProtocolVersions {
+		requestHeader.Add(ProtocolAcceptVersionHeader, supportedProtocolVersion)
+	}
+	// for backward compatability, include the ProtocolVersion header as well.
+	requestHeader.Set(ProtocolVersionHeader, ProtocolVersion)
```

Comment:
Doesn't  ProtocolAcceptVersionHeader implies own version?
What kind of backward compatibility?

Reply:
Existing relays expect to get the ProtocolVersionHeader.

Reply:
Existing relays expect to get the ProtocolVersionHeader.